### DISCUSSION TITLE
哟西，这里可真是个好地方

### DISCUSSION THREAD

**Discussion by wum2008** [share121/inter-knot#1989] [其他]
![IMG_20240725_230029](https://github.com/user-attachments/assets/f3196134-1243-4480-b971-2a50add24870)

**Comment by wum2008** [comment]
这里等级怎么升呀

**Comment by Dave-12138** [comment]
> 这里等级怎么升呀

等级其实就是你的公开仓库(repositories)数

**Reply by wum2008** [reply]
搜嘎，谢谢
